bin/upgrade-hobby

```diff
@@ -11,31 +11,41 @@ else
     exit
 fi
 
-echo ""
-echo ""
-echo "🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨"
-echo "🚨🚨🚨🚨🚨 WARNING: POTENTIAL DATA LOSS 🚨🚨🚨🚨🚨🚨"
-echo "🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨"
-echo ""
-echo ""
-echo "PRIOR TO THIS VERISON, the Postgres and Clickhouse containers did NOT have persistent named volumes."
-echo "If you choose to upgrade, you 💣 may lose data 💣 contained in anonymous volumes."
-echo ""
-echo "See the discussion here for more information: https://github.com/PostHog/posthog/pull/11256"
-echo ""
-echo "WE STRONGLY RECOMMEND YOU:"
-echo ""
-echo "✅ back up your entire environment/installation (vm, host, etc.), including all docker containers and volumes:"
-echo "✅ specifically back up the contents of :"
-echo "  ☑ /var/lib/postgresql/data in the postgres (root_db_1) container"
-echo "  ☑ /var/lib/clickhouse in the root_clickhouse_1 (root_clickhouse_1) container"
-echo "and be ready to check/recopy the data before you boot Posthog next."
-read -r -p "Do you want to upgarde PostHog? [y/N] " response
-if [[ "$response" =~ ^([yY][eE][sS]|[yY])+$ ]]
+echo "Checking for named postgres and clickhouse volumes to avoid data loss when upgrading from < 1.39"
+if docker volume ls | grep -Pzoq 'root_clickhouse-data\n(.|\n)*root_postgres-data\n'
```

Comment:
hmm I have a feeling that the `root_` prepend is directory dependeny @danielthedifficult , am I wrong? I haven't checked. If not then all good.